greymatter-io/gitops-core
# 0.11.0 release activities

[sc-28120]

- Bump default operator version to 0.11.0
- Regenerate operator.yaml
- Add an `operator_withspire.yaml` with the same content plus the spire manifests. (The difference during generation is `-t spire=true` on the `cue eval`.) BTW, manual operations performed were a) delete the gm-operator namespace creation from the manifests, and b) change `greymatter-io` to `<your-org>` to match the docs.

## How to test

- You should be able to follow the docs to install operator 0.11.0
- Also try installing with spire by deviating from the docs in two ways:
    - Set `spire` to `true` and push that change to your gitops target repository.
    - Use `operator_withspire.yaml` instead of `operator.yaml`
